app/views/password_reset_mailer/password_reset.text.erb

```diff
@@ -0,0 +1,14 @@
+Hello <%= @user.first_name %>,
```

Comment:
Any chance any of these values could be nil? The email will read oddly if they are.